src/CTA.Rules.Actions/ProjectLevelActions.cs

```diff
@@ -88,5 +89,31 @@ static string func(string projectDir, ProjectType projectType)
 
             return func;
         }
+
+        public Func<string, ProjectType, string> GetCreateMonolithServiceAction(string namespaceString)
+        {
+            string func(string projectDir, ProjectType projectType)
+            {
+                if(projectDir != null && projectDir.Length > 0)
+                {
+                    projectDir = projectDir.Substring(0, projectDir.LastIndexOf("\\") + 1);
```

Comment:
Can we use Path.GetDirectoryName (or something similar) instead?

Reply:
Agreed and fixed thanks.